src/server/browserContext.ts

```diff
@@ -283,6 +284,11 @@ export abstract class BrowserContext extends SdkObject {
         // Close all the pages instead of the context,
         // because we cannot close the default context.
         await Promise.all(this.pages().map(page => page.close(metadata)));
+        if (this._browser.options.isChromium) {
+          await Promise.all([...(this._browser as CRBrowser)._backgroundPages.entries()].map(
```

Comment:
I am a bit afraid this is going to stall for some reason. Extensions coverage is abysmal... Can we clear the list of pages on the client after closure? Or just pretend that they did all close, like we do in `_browserClosed`?

Reply:
Updated it, now it should be faked and also safe to call when it's done multiple times.